# Proyecto Final
by Matías Moré

In [1]:
#Librerias
import pandas as pd
import numpy as np


## Dataset Fumadores

Este conjunto de datos consiste en datos biologicos de distintos individuos, recolectados con el objetivo de analizar los efectos y las señales del habito tabaquico. A travez de distintas caracteristicas de estudios de laboratorio como el estado sanguineo, urinario, hepatico, higienico y su relacion con el tabaquismo, se buscara encontrar patrones que determinen biologicamente los parametros a tener en cuenta para identificar a individuos fumadores. Teniendo en cuenta estos datos se realizara una comparacion de estas variables a fin de determinar su utilidad y trascendencia para lograr este fin.

Origen del dataset: https://www.kaggle.com/datasets/kukuroo3/body-signal-of-smoking?resource=download

### Code Acquisition

In [3]:
smoke_data = pd.read_csv('/workspaces/CoderPosta/Data/smoking.csv')
smoke_data

ID gender  age  height(cm)  weight(kg)  waist(cm)  eyesight(left)  \
0          0      F   40         155          60       81.3             1.2   
1          1      F   40         160          60       81.0             0.8   
2          2      M   55         170          60       80.0             0.8   
3          3      M   40         165          70       88.0             1.5   
4          4      F   40         155          60       86.0             1.0   
...      ...    ...  ...         ...         ...        ...             ...   
55687  55676      F   40         170          65       75.0             0.9   
55688  55681      F   45         160          50       70.0             1.2   
55689  55683      F   55         160          50       68.5             1.0   
55690  55684      M   60         165          60       78.0             0.8   
55691  55691      M   55         160          65       85.0             0.9   

       eyesight(right)  hearing(left)  hearing(right)  ...  hemoglobin  \
0                  1.0            1.0             1.0  ...        12.9   
1                  0.6            1.0             1.0  ...        12.7   
2                  0.8            1.0             1.0  ...        15.8   
3                  1.5            1.0             1.0  ...        14.7   
4                  1.0            1.0             1.0  ...        12.5   
...                ...            ...             ...  ...         ...   
55687              0.9            1.0             1.0  ...        12.3   
55688              1.2            1.0             1.0  ...        14.0   
55689              1.2            1.0             1.0  ...        12.4   
55690              1.0            1.0             1.0  ...        14.4   
55691              0.7            1.0             1.0  ...        15.0   

       Urine protein  serum creatinine   AST   ALT   Gtp  oral  dental caries  \
0                1.0               0.7  18.0  19.0  27.0     Y              0   
1                1.0               0.6  22.0  19.0  18.0     Y              0   
2                1.0               1.0  21.0  16.0  22.0     Y              0   
3                1.0               1.0  19.0  26.0  18.0     Y              0   
4                1.0               0.6  16.0  14.0  22.0     Y              0   
...              ...               ...   ...   ...   ...   ...            ...   
55687            1.0               0.6  14.0   7.0  10.0     Y              1   
55688            1.0               0.9  20.0  12.0  14.0     Y              0   
55689            1.0               0.5  17.0  11.0  12.0     Y              0   
55690            1.0               0.7  20.0  19.0  18.0     Y              0   
55691            1.0               0.8  26.0  29.0  41.0     Y              0   

       tartar  smoking  
0           Y        0  
1           Y        0  
2           N        1  
3           Y        0  
4           N        0  
...       ...      ...  
55687       Y        0  
55688       Y        0  
55689       N        0  
55690       N        0  
55691       Y        1  

[55692 rows x 27 columns]

## Dataset Causas de Muerte

El archivo de datos presentado contiene datos históricos sobre las diferentes causas de muerte en todas las edades en todo el mundo. Los principales aspectos del conjunto de datos son: meningitis, enfermedad de Alzheimer y otras demencias, enfermedad de Parkinson, carencias nutricionales, malaria, ahogamiento, violencia interpersonal, trastornos maternos, VIH / SIDA, trastornos relacionados con el uso de drogas, tuberculosis, enfermedades cardiovasculares, infecciones respiratorias bajas, trastornos neonatales, trastornos relacionados con el uso de alcohol, auto daño, exposición a las fuerzas de la naturaleza, enfermedades diarreicas, exposición al calor y frío ambiental, neoplasias, conflicto y terrorismo, diabetes mellitus, enfermedad renal crónica, envenenamientos, malnutrición proteico-energética, lesiones en carretera, enfermedades respiratorias crónicas, cirrosis y otras enfermedades hepáticas crónicas, enfermedades digestivas, fuego, calor y sustancias calientes, hepatitis aguda.
Este conjunto de datos permite detectar tendencias en las causas de muerte a nivel mundial, utilizando metricas como DALY (Años de Vida Ajustados por Discapacidad), podriamos comparar la carga de enfermedad de diferentes enfermedades en diferentes países, entre diferentes poblaciones y en diferentes momentos.

Origen del database: https://www.kaggle.com/datasets/iamsouravbanerjee/cause-of-deaths-around-the-world

### Code Acquisition

In [4]:
cod_data = pd.read_csv('/workspaces/CoderPosta/Data/cause_of_deaths.csv')
cod_data

Country/Territory Code  Year  Meningitis  \
0          Afghanistan  AFG  1990        2159   
1          Afghanistan  AFG  1991        2218   
2          Afghanistan  AFG  1992        2475   
3          Afghanistan  AFG  1993        2812   
4          Afghanistan  AFG  1994        3027   
...                ...  ...   ...         ...   
6115          Zimbabwe  ZWE  2015        1439   
6116          Zimbabwe  ZWE  2016        1457   
6117          Zimbabwe  ZWE  2017        1460   
6118          Zimbabwe  ZWE  2018        1450   
6119          Zimbabwe  ZWE  2019        1450   

      Alzheimer's Disease and Other Dementias  Parkinson's Disease  \
0                                        1116                  371   
1                                        1136                  374   
2                                        1162                  378   
3                                        1187                  384   
4                                        1211                  391   
...                                       ...                  ...   
6115                                      754                  215   
6116                                      767                  219   
6117                                      781                  223   
6118                                      795                  227   
6119                                      812                  232   

      Nutritional Deficiencies  Malaria  Drowning  Interpersonal Violence  \
0                         2087       93      1370                    1538   
1                         2153      189      1391                    2001   
2                         2441      239      1514                    2299   
3                         2837      108      1687                    2589   
4                         3081      211      1809                    2849   
...                        ...      ...       ...                     ...   
6115                      3019     2518       770                    1302   
6116                      3056     2050       801                    1342   
6117                      2990     2116       818                    1363   
6118                      2918     2088       825                    1396   
6119                      2884     2068       827                    1434   

      ...  Diabetes Mellitus  Chronic Kidney Disease  Poisonings  \
0     ...               2108                    3709         338   
1     ...               2120                    3724         351   
2     ...               2153                    3776         386   
3     ...               2195                    3862         425   
4     ...               2231                    3932         451   
...   ...                ...                     ...         ...   
6115  ...               3176                    2108         381   
6116  ...               3259                    2160         393   
6117  ...               3313                    2196         398   
6118  ...               3381                    2240         400   
6119  ...               3460                    2292         405   

      Protein-Energy Malnutrition  Road Injuries  \
0                            2054           4154   
1                            2119           4472   
2                            2404           5106   
3                            2797           5681   
4                            3038           6001   
...                           ...            ...   
6115                         2990           2373   
6116                         3027           2436   
6117                         2962           2473   
6118                         2890           2509   
6119                         2855           2554   

      Chronic Respiratory Diseases  \
0                             5945   
1                             6050   
2                             6223   
3                             6445   
4                 

## Dataset Juegos Steam

Esta base de datos contiene información sobre las páginas de los productos de todos los 55691 juegos (sin contar DLC y otros contenidos secundarios) en la tienda en línea de juegos para PC Steam. Fue recolectado el 8 de noviembre de 2022. Incluye información valiosa como el género, las etiquetas, las categorías, los idiomas y el número de reseñas positivas y negativas de cada juego. Este conjunto de datos permite identificar tendencias en el mercado de videojuegos y desarrollar juegos acordes a estas tendencias, lo que lo vuelve una potencial herramienta de desarrollo para empresas independientes asi como tambien las grandes empresas.

Origen del database: https://www.kaggle.com/datasets/tristan581/all-55000-games-on-steam-november-2022

### Code Acquisition

In [4]:
steam_data = pd.read_csv('/workspaces/CoderPosta/Data/steam_games.csv')
steam_data

ParserError: Error tokenizing data. C error: Expected 37 fields in line 4, saw 40
